flexget/plugins/input/next_trakt_episodes.py

```diff
@@ -75,6 +75,7 @@ def on_task_input(self, task, config):
                     trakt_id = item['show']['ids']['trakt']
                     listed_series[trakt_id] = {
                         'series_name': '%s (%s)' % (item['show']['title'], item['show']['year']),
+                        'series_year': item['show']['year'],
```

Comment:
I get your intentions, but as we've said, it's for historical reasons. Doesn't mean you don't have a valid point, it's just not something that belongs in this PR.